app/views/root/designprinciples.html.erb

```diff
@@ -1,145 +1,68 @@
 <% content_for :title do %>GOV.UK – GDS design principles<% end %>
 
- <div id="wrapper" class="design-principles"> 
-  <div class="beta-notice">
-    <div class="inner">
-    <p>
-    <a href="http://digital.cabinetoffice.gov.uk/2012/04/03/introducing-the-design-principles-alpha-for-gds/">ALPHA</a> <span>Last updated 2 July 2012</span>
-    </p>
-    </div>
-  </div>
-  <header>
+ <div id="wrapper" class="design-principles">
+  <header class="gds-principles">
     <h1>Government Digital Service <strong>Design Principles</strong></h1>
     <p>Listed below are our design principles and examples of how we’ve used them so far. These build on, and add to, our original <a href="http://www.flickr.com/photos/benterrett/7041509709/">7 digital principles</a>.</p>
```

Comment:
It is a thing we should do yes, but I don't think it is worth holding up this change to make it.